packages/plugin-ext/package.json

```diff
@@ -25,7 +25,11 @@
     "@theia/terminal": "^0.9.0",
     "@theia/workspace": "^0.9.0",
     "@types/request": "^2.0.3",
+    "@types/decompress": "^4.2.2",
+    "@types/escape-html": "^0.0.20",
+    "@types/lodash.clonedeep": "^4.5.3",
```

Comment:
Why types are dependencies ?

Reply:
Request from @kittaakos, I would like to know why as well :)

Reply:
I could not find the thread I was looking for; please proceed without the typings.